docs/pt/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -0,0 +1,122 @@
+
+Passe `*`, como o primeiro parâmetro da função.
+
+O Python não vai fazer nada com esse `*`, mas ele vai saber que a partir dali os parâmetros seguintes deverão ser chamados argumentos nomeados (pares chave-valor), também conhecidos como <abbr title="Do inglês: K-ey W-ord Arg-uments"><code>kwargs</code></abbr>. Mesmo que eles não possuam um valor padrão.
+
+```Python hl_lines="8"
+{!../../../docs_src/path_params_numeric_validations/tutorial003.py!}
+```
+
+## Validações numéricas: maior que ou igual
+
+Com `Query` e `Path` (e outras que você verá mais tarde) você pode declarar restrições de string, e também restrições numéricas.
+
+Aqui, com `ge=1`, `item_id` precisará ser um numéro inteiro maior que ("`g`reater than") ou igual ("`e`qual") a 1.
+
+```Python hl_lines="8"
+{!../../../docs_src/path_params_numeric_validations/tutorial004.py!}
+```
+
+## Validações numéricas: maior que e menor que ou igual
+
+O mesmo se aplica para:
+
+* `gt`: maior que (`g`reater `t`han)
+* `le`: menor que ou igual (`l`ess than or `e`qual)
+
+```Python hl_lines="9"
+{!../../../docs_src/path_params_numeric_validations/tutorial005.py!}
+```
+
+## Validações numéricas: valores do tipo float, maior que e menor que
+
+Validações numéricas também funcionam para valores do tipo `float`.
+
+Aqui é onde se torna importante a possibilidade de declarar <abbr title="greater than"><code>gt</code></abbr> e não apenas <abbr title="greater than or equal"><code>ge</code></abbr>. Com isso você pode especificar, por exemplo, que um valor deve ser maior que `0`, ainda que seja menor que `1`.
+
+Assim, `0.5` seria um valor válido. Mas `0.0` ou `0` não seria.
+
+E o mesmo para <abbr title="less than"><code>lt</code></abbr>.
+
+```Python hl_lines="11"
+{!../../../docs_src/path_params_numeric_validations/tutorial006.py!}
+```
+
+## Recapitulando
+
+Com `Query`, `Path` (e outras que você ainda não viu) você pode declarar metadados e validações de texto do mesmo modo que com [Parâmetros de consulta e validações de texto](query-params-str-validations.md){.internal-link target=_blank}.
+
+E você também pode declarar validações numéricas:
+
+* `gt`: maior que (`g`reater `t`han)
+* `ge`: maior que ou igual (`g`reater than or `e`qual)
+* `lt`: menor que (`l`ess `t`han)
+* `le`: menor que ou igual (`l`ess than or `e`qual)
+
+!!! info
+    `Query`, `Path`, e outras que você verá a frente são subclasses de uma classe `Param` comum (que você não precisa usar).
+
+    E todas elas compartilham todos esses mesmos parâmetros de validação adicional e metadados que você viu.
+
+!!! note "Detalhes Técnicos"
+    Quando você importa `Query`, `Path` e outras de `fastapi`, elas são na verdade funções.
+
+    Que quando chamadas, retornam instâncias de classes de mesmo nome.
+
+    Então, você importa `Query`,que é uma função. E quando você a chama, ela retorna uma instância de uma classe também chamada `Query`.
```

Comment:
```suggestion
    Então, você importa `Query`, que é uma função. E quando você a chama, ela retorna uma instância de uma classe também chamada `Query`.
```

Reply:
Thanks for the suggestions @LorhanSohaky , little details that make the text better to read. I will put the suggestions in a new commit.